backend/cesium_api/app.py

```diff
@@ -14,11 +15,12 @@ class App(Flask):
 
     def __init__(
         self: App,
-        data: Data,
+        data: Data["DATA"],
```

Comment:
```suggestion
        data: Data,
```

What's the purpose of `["DATA"]`? The `Data` class itself is not index-able so this would not hint the right type.